polaris-react/src/components/Grid/components/Cell/Cell.tsx

```diff
@@ -31,13 +31,15 @@ export interface CellProps {
    * https://polaris.shopify.com/components/layout-and-structure
    */
   area?: string;
+  colStart?: Cell;
```

Comment:
What do you think about removing or spliting this addition off to a separate PR, then review the other MDX component changes (`Small`, `Medium`, etc)?